perf/gh-7089-vclock-copy/Makefile

```diff
@@ -0,0 +1,21 @@
+test: test_none stop
+
+test_wal: test_write stop
+
+dirs:
+	for i in {1..11}; do mkdir $${i} || true; done
+
+start_swarm:
+	for i in {2..11}; do ./swarm.lua $${i}; done
+
+test_none: clean dirs start_swarm
+	./speedtest.lua none
```

Comment:
Maybe pass the working directory explicitly here as for `swarm.lua`? It is not obvious from the Makefile how the `1/` directory is used.

Reply:
Ok.